app/move/controllers/create/index.js

```diff
@@ -1,13 +1,15 @@
 const Assessment = require('./assessment')
 const Base = require('./base')
 const MoveDetails = require('./move-details')
+const MoveCompleted = require('./move-completed')
```

Comment:
Wonder if it would be better to call it a _confirmation_ page rather than _completed_ in hindsight.

We will be using the term "completed" for a different action at some point - when the move from A to B has been fully completed.

This is more of a confirmation of the _request_ being booked, which annoyingly also doesn't guarantee it because the supplier could still choose not to move the person (which is basically the cancellation work I'm doing).